src/features/rewards/tipsMigrationAlert/index.tsx

```diff
@@ -3,28 +3,112 @@
 * You can obtain one at http://mozilla.org/MPL/2.0/.
 */
 
+// Utils
 import * as React from 'react'
 import { getLocale } from '../../../helpers'
 
+// Components
 import {
   StyledWrapper,
   StyledAlertIcon,
   StyledInfo,
   StyledMessage,
   StyledMonthlyTips,
   StyledReviewWrapper,
-  StyledReviewList
+  StyledReviewList,
+  StyledModalContent,
+  StyledModalInfo,
+  StyledListMessage,
+  StyledList,
+  StyledListItem,
+  StyledTipsIcon,
+  StyledButton
 } from './style'
+import { Modal } from '../../../components'
 import { AlertCircleIcon } from '../../../components/icons'
+import Button from '../../../components/buttonsIndicators/button'
+
+// Assets
+const sendTipsImage = require('./assets/sendTips')
```

Comment:
please use import here and export default in image file, so that is inlined with @petemill work